test.properties

```diff
@@ -122,7 +122,7 @@
     browser.type[tck]=chrome
 
     #browser.chrome.bin.args=
-    #browser.chrome.bin.file=
+    browser.chrome.bin.file=C:/chrome-win/chrome.exe
```

Comment:
Please **DO NOT** edit and commit this file. You can edit `test.username.properties` file instead.